### ISSUE TITLE
Column macro

### ISSUE THREAD

**Issue by mandeep** [phsym/prettytable-rs#42]
Hi @phsym,

I stumbled upon your crate for an application I'm writing and noticed that there isn't an easy way to create a table from columnar data. Have I missed this in the documentation, or is this yet to be implemented?

Thanks!
Mandeep

**Comment by phsym** [comment]
Hi Mandeep,
What do you mean exactly ? Are you try to create a table from a list of columns instead of a list of rows ?

**Comment by mandeep** [comment]
Yep exactly that. Currently, I'm iterating through all of my vectors of data and adding the data to rows in the table. I'm not sure there is a time penalty for this, but I think being able to create columns would be a nice feature.

**Comment by phsym** [comment]
There's currently no way to do that in the library. But it could be nice indeed. I'm not sure there is a time penalty with what you do, since the `Table` struct is backed by a vector of rows. A methods to import from column data would probably do it the same way you do

**Comment by danieleades** [comment]
this would be super useful.

Consider the case that you wish to display a table of data to the user, but allow them to specify which fields (columns, equivalently) to display. what would be the idiomatic way to construct a table such as this using this library?

Being able to construct a table by columns would make this trivial.

Perhaps a `Vec<Row>` type is not the best representation to use. Can `ndarray` offer a better solution?
